nodeup/pkg/model/kube_apiserver.go

```diff
@@ -71,7 +71,7 @@ func (b *KubeAPIServerBuilder) Build(c *fi.ModelBuilderContext) error {
 			}
 
 			key := "encryptionconfig"
-			encryptioncfg, _ := b.SecretStore.Secret(key)
+			encryptioncfg, err := b.SecretStore.Secret(key)
 			if encryptioncfg != nil {
```

Comment:
This seems odd to read the error, but condition based on the `encryptioncfg` value.

Reply:
Yeah, you are right. If the secret is not set, `Secret` will return an error too. So I guess we can trust we have a secret if `err` is `nil`. Amended.